### PR TITLE
Peripherals: added SmartAP GPS page

### PR DESCRIPTION
**Pull request by aviaks**
Added SmartAP GPS page

### CODE REVIEW

File: en/gps_compass/gps_smartap.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Kit Contents
+
+An SmartAP GPS kit includes:
+- 1x GPS modules
+- 1x 30cm cable
+
+## Configuration
+
+For the aircraft, you should set the parameter [SER_GPS1_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD) to 115200 8N1 to ensure that PX4 configures the correct baudrate.
```
Comment (hamishwillee) [inline en/gps_compass/gps_smartap.md:27]:
```suggestion
For the aircraft, you should set the parameter [SER_GPS1_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD) to 115200 8N1 to ensure that PX4 uses the correct baudrate.
```

File: en/gps_compass/gps_smartap.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pinout and Connections
+
+SmartAP GPS has 10 pin JST-GH connector that can be plugged into a PixHawk autopilot.
```
Comment (hamishwillee) [inline en/gps_compass/gps_smartap.md:31]:
```suggestion
SmartAP GPS has 10 pin JST-GH connector that can be plugged into a Pixhawk flight controller (compliant with Pixhawk connector standard).
```

**Review by hamishwillee — APPROVED** [review]
Thank you. Really nice to review something where there is nothing for me to do!

**Comment by aviaks** [comment]
@hamishwillee awesome, appreciate it! This is thanks to your great work with creating documentation guidelines!

**Pull request merged**